homeassistant/components/atag/strings.json

```
@@ -0,0 +1,21 @@
+{
+  "config": {
+    "title": "Atag",
+    "step": {
+      "user": {
+        "title": "Connect to the device",
+        "data": {
+          "host": "Host",
+          "email": "Email (optional)",
```

```suggestion
          "email": "Email (optional)",
```
Looks unused, please remove